### DISCUSSION TITLE
Question:

### DISCUSSION THREAD

**Discussion by hehehehehehehehehehehehehehehehehehe** [FunkinCrew/Funkin#1516] [Q&A] [answered]
im confused, where is, "the root of the project," i dont know where to find it and i want to compile the game

**Comment by ghost** [comment]
I'm not sure but I think it's project.xml

**Comment by Two-Ai** [comment, marked as answer]
The "project root" is the top-level directory (probably called `Funkin` if you cloned it from GitHub). In other words, the folder that contains `Project.xml`, `README.md` and all the other files and folders.
